amida-tech/amida-messaging-microservice
# Feature/add UUID

If this PR fixes a bug, you _must_ add test cases representative of the bug.

#### What's this PR do?

Check if uuids are missing from user documents. Add uuid for users, where missing, from auth service. If one does not exist on auth, one will be generated by orange-api. 
 
#### Related JIRA tickets:

- https://jira.amida.com/browse/ORANGE-481

#### How should this be manually tested?

Run `yarn && yarn start`. You should run this *AFTER* you have already started the auth service on the branch that has uuid support. 

#### Any background context you want to provide?

Support new microservices using uuid versus username (which is typically registration email)
